test.js

```diff
@@ -1,77 +1,33 @@
 import { equal } from 'assert';
 import mdast from 'mdast';
-import typographer from './';
+import base from 'typographic-base';
+import typographer from './index';
 
 const t = (text, options) =>
   mdast().use(typographer, options).process(text).trim();
```

Comment:
I would suggest this API
```
import mdast from 'mdast';
import mdtextr from './index';
import base from 'typographic-base';

mdast()
  .use(mdtextr, {
    plugins: [ base ]
    options: { locale: 'uk' }
  })
  .process(`"quotes"`); 
```

suggested way is _convenient_ with current textr’s state and it’s much _easier to describe (and to understand)_ in the readme